src/main/java/io/polivakha/mojo/properties/models/StoreProperties.java

```diff
@@ -0,0 +1,44 @@
+package io.polivakha.mojo.properties.models;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Enumeration;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class StoreProperties extends Properties {
```

Comment:
I think it is too much of overcomplication. You created a separate class just to sort properties before writing to disk. It is not even clear how overridden methods should help you here. this is clear only when you know the implementation details of `Properties#store` method, which breaks encapsulation idea.

```
new Properties(new TreeMap<>(properties)).store(outputStream, null)
```

something like that would be suffice.